### PR TITLE
Create disk_element.py

### PR DESCRIPTION
**Pull request by rodrigomoliveira1**
This file creates disk elements

### CODE REVIEW

File: disk_element.py [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+
+    def summary(self):
+        """A summary for the element.
+        A pandas series with the element properties as variables.
+        """
+        attributes = self.__dict__
+        attributes["type"] = self.__class__.__name__
+        return pd.Series(attributes)
+
+class LumpedDiskElement(Element):
```
Comment (raphaeltimbo) [inline disk_element.py]: We will have only one class for the disks with one additional constructor for geometry arguments:

```python
class DiskElement(Element):
    def __init__(self, m, Id, Ip):
        ...
    
    @classmethod
    def from_geometry(cls, n, material, width, i_d, o_d):
        ...
        return cls(n, m, Id, Ip)
```

File: disk_element.py [hunk 2]
```diff
@@ -0,0 +1,336 @@
+import warnings
+import numpy as np
+import pandas as pd
+import scipy.interpolate as interpolate
+import matplotlib.pyplot as plt
+import matplotlib.patches as mpatches
+from ross.data_io.read_xl import (
```
[the remark this answers is not in the document]
Comment (rodrigomoliveira1) [reply disk_element.py]: Changes are done. The following commit is [here](https://github.com/rodrigomoliveira1/ross/commit/3dd5aa3eca6161ae28f2c31d59c1e067219716a6).

File: disk_element.py [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+        w_range = np.linspace(min(self.w), max(self.w), 30)
+
+        ax.plot(w_range, self.interpolated(w_range), **kwargs)
+        ax.set_xlabel("Speed (rad/s)")
+        ax.ticklabel_format(style="sci", scilimits=(0, 0), axis="y")
+
+        return ax
+
+
+class _Stiffness_Coefficient(_Coefficient):
```
[the remark this answers is not in the document]
Comment (rodrigomoliveira1) [reply disk_element.py]: Changes are done. The following commit is [here](https://github.com/rodrigomoliveira1/ross/commit/eca5accb3bc2d9b9f21fd2f395e0692acc041c30).

File: disk_element.py [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+            [zpos + hw, -(ypos + 0.9 * D)],
+            [zpos - hw, -(ypos + 0.9 * D)],
+            [zpos - hw, -(ypos + 0.1 * D)],
+            [zpos, -ypos],
+        ]
+        ax.add_patch(mpatches.Polygon(disk_points_u, facecolor=self.color))
+        ax.add_patch(mpatches.Polygon(disk_points_l, facecolor=self.color))
+
+
+class _Coefficient:
```
[the remark this answers is not in the document]
Comment (rodrigomoliveira1) [reply disk_element.py]: Changes are done. The following commit is [here](https://github.com/rodrigomoliveira1/ross/commit/da501b3b72b7db6200bf104fe0b0680ac9fbde1d).

**Pull request merged**